flocxclient/common/apiclient/base.py

```diff
@@ -0,0 +1,27 @@
+"""
```

Comment:
We should probably add a license header - let's use whatever I used for esi-leap:

https://github.com/CCI-MOC/esi-leap/blob/master/esi_leap/api/app.py

Reply:
Would this be added to every file? Every file save for __init__.py files have this exact header

Reply:
Yep, every *,py file except for init.py.